cves/kernel/CVE-2021-3659.yml

```diff
@@ -335,9 +340,9 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: true
+  stacktrace_with_fix: false
+  note: "There was a call trace, which I'm assuming is close enough"
```

Comment:
Remove the part after the comma and mention that the call trace was in the commit message of the fix.